packages/expo-modules-core/src/uuid/uuid.ts

```diff
@@ -14,8 +14,25 @@ function uuidv4(): string {
   return nativeUuidv4();
 }
 
+function uuidv5(name: string, namespace: string | number[]) {
+  const parsedNamespace =
+    Array.isArray(namespace) && namespace.length === 16 ? bytesToUuid(namespace) : namespace;
+
+  // If parsed namespace is still an array it means that it wasn't valid
+  if (Array.isArray(parsedNamespace)) {
+    throw new Error('`namespace` must be uuid string or an Array of 16 byte values');
```

Comment:
We should use consistent casing in error messages (`uuid` -> `UUID`)
```suggestion
    throw new Error('`namespace` must be a valid UUID string or an Array of 16 byte values');
```